### PR TITLE
chore(schema): reporting/model stack supports timezone setting for app

### PR DESCRIPTION
**Pull request by llmin**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

## Summary

(describe what this merge request does)

## Implementation highlights

(describe how the merge request does for feature changes, share the RFC link if it has)

## Test checklist

- [x] add new test cases
- [x] all code changes are covered by unit tests
- [ ] end-to-end tests
  - [ ] deploy web console with CloudFront + S3 + API gateway
  - [ ] deploy web console within VPC
  - [ ] deploy ingestion server
    - [ ] with MSK sink
    - [ ] with KDS sink
    - [ ] with S3 sink
  - [ ] deploy data processing
  - [ ] deploy data modeling
    - [ ] new Redshift Serverless
    - [ ] provisioned Redshift
    - [ ] Athena
  - [x] deploy with reporting
  - [ ] streaming ingestion
    - [ ] with Redshift Serverless
    - [ ] with provisioned Redshift

## Is it a breaking change

- [ ] add parameters without default value in stack
- [ ] introduce new service permission in stack
- [ ] introduce new top level stack module

## Miscellaneous

- [ ] introduce new symbol link source file(s) to be shared among infra code, web console frontend, and web console backend

### CODE REVIEW

File: src/reporting/parameter.ts [hunk 1]
```diff
@@ -80,15 +80,6 @@ export function createStackParametersQuickSight(scope: Construct, paramGroups?:
     default: 'QuickSight Owner Principal Arn',
   };
 
-  const quickSightPrincipalParam = new CfnParameter(scope, 'QuickSightPrincipalParam', {
```
Comment (zxkane) [inline src/reporting/parameter.ts:83]: @tyyzqmf Is it safe to create/upgrade to the latest stack without this parameter?

File: src/reporting/quicksight-custom-resource.ts [hunk 2]
```diff
@@ -87,8 +87,8 @@ export function createQuicksightCustomResource(
 
   const eventViewColumns = `
     *, 
-    DATE_TRUNC('second', CONVERT_TIMEZONE('${props.timezone}', event_timestamp)) ::timestamp AS event_timestamp_local,
-    DATE_TRUNC('day', CONVERT_TIMEZONE('${props.timezone}', event_timestamp)) ::timestamp AS event_date
+    DATE_TRUNC('second', CONVERT_TIMEZONE('{{{timezone}}}', event_timestamp)) ::timestamp AS event_timestamp_local,
```
Comment (zxkane) [inline src/reporting/quicksight-custom-resource.ts:90]: Will it be replaced by QuickSight?

File: src/reporting/parameter.ts [hunk 3]
```diff
@@ -99,15 +90,14 @@ export function createStackParametersQuickSight(scope: Construct, paramGroups?:
   };
 
   const quickSightTimezoneParam = new CfnParameter(scope, 'QuickSightTimezoneParam', {
-    description: 'Timezone of the builtin dashboard.',
+    description: 'The time zone with app id',
     type: 'String',
-    default: 'UTC',
+    default: '[]',
```
Comment (zxkane) [inline src/reporting/parameter.ts:95]: @tyyzqmf Is it same format with data modeling?

File: src/reporting/parameter.ts (hunk 1 above)
Comment (llmin) [reply src/reporting/parameter.ts:83]: @tyyzqmf already completed completed the corresponding modifications。

File: src/reporting/quicksight-custom-resource.ts (hunk 2 above)
Comment (llmin) [reply src/reporting/quicksight-custom-resource.ts:90]: no. this just used in dataset custom sql definition.  already checked.

Comment (llmin) [reply src/reporting/quicksight-custom-resource.ts:90]: used in date set custom sql definition. already checked it is right.  We need check whether is it right at visual render time.

File: src/reporting/parameter.ts (hunk 3 above)
Comment (llmin) [reply src/reporting/parameter.ts:95]: confirmed with @tyyzqmf , It is same format with modeling.

**Review by zxkane — APPROVED** [review]

**Review by zxkane — APPROVED** [review]

**Pull request merged**